frontend/server/src/Controllers/Course.php

```diff
@@ -2338,7 +2338,52 @@ public static function apiAssignmentDetails(\OmegaUp\Request $r) {
             $problem['letter'] = \OmegaUp\Controllers\Contest::columnName(
                 $letter++
             );
+            $problem['quality_payload'] = [
+                'solved' => false,
+                'tried' => false,
+                'nominated' => false,
+                'dismissed' => false,
+            ];
+            $problemObject = \OmegaUp\DAO\Problems::getByPK(
+                $problem['problem_id']
+            );
+            if (is_null($problemObject)) {
+                throw new \OmegaUp\Exceptions\NotFoundException(
+                    'problemNotFound'
+                );
+            }
+            $nominationStatus = \OmegaUp\DAO\QualityNominations::getNominationStatusForProblem(
+                $problemObject,
+                $r->identity
+            );
+            $nominationStatus['tried'] = false;
+            $nominationStatus['solved'] = false;
+            $details = \OmegaUp\Controllers\Problem::getProblemDetails(
+                $r,
+                $problemObject,
+                null,
+                '',
+                false
+            );
+            foreach ($details['runs'] ?? [] as $run) {
+                if ($run['verdict'] === 'AC') {
+                    $nominationStatus['solved'] = true;
+                    break;
+                } elseif ($run['verdict'] !== 'JE' && $run['verdict'] !== 'CE') {
+                    $nominationStatus['tried'] = true;
+                }
```

Comment:
`tried` no se va a establecer si el primer intento fue `AC`. qué te parece si mejor cambias el orden?

```suggestion
                if ($run['verdict'] !== 'JE' && $run['verdict'] !== 'CE') {
                    $nominationStatus['tried'] = true;
                }
                if ($run['verdict'] === 'AC') {
                    $nominationStatus['solved'] = true;
                    break;
                }
```

Reply:
Listo